layouts/default.vue

```diff
@@ -1,9 +1,15 @@
-<script setup>
+<script setup lang="ts">
 import { ref, onMounted } from 'vue'
 import { useRuntimeConfig } from '#app'
 const config = useRuntimeConfig()
 const route = useRoute()
 const atTop = ref(true)
+const { data:navigationData } = await findNavigation()
```

Comment:
This might be simpler to read like `const navigation = await findNavigation().then({data} => normalizeFindNavigationResponse(data)); This follows for the same few lines.

Reply:
Went back and forth on whether this was really simpler but I think you're right.